README.aspnetcore.md

```diff
@@ -15,10 +15,14 @@ These images contain the runtime only. Use [`microsoft/aspnetcore-build-nightly`
 - `1.1.2`, `1.1`, `1`
     - [`1.1.2-jessie` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/1.1/jessie/runtime/Dockerfile)
     - [`1.1.2-nanoserver` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/1.1/nanoserver/runtime/Dockerfile)
-- `2.0.0`, `2.0`, `2`, `latest`
+- `2.0.0`, `2.0`
```

Comment:
Ah yes - sorry about that - I needed to make a schema change in order to support ARM.  [Here ](https://github.com/dotnet/dotnet-docker-nightly/commit/a1dcba51294abadf17a07c353e3586c464984143#diff-4b1eb3dc48c4e16d49db5b42298fe654L24) is a sample of how you would need to change the schema.